### PR TITLE
Migrate cron (send old issues and PR details to slack) to GitHub actions

### PR DESCRIPTION
**Pull request by rejahrehim**
This PR is part of  issue #340

- [x] You verified/tested the effectiveness of your contribution (e.g., the defensive code proposed is really an effective remediation? Please verify it works!).
- [x] The CI build of your PR pass

@ThunderSon , Could you please add a new secret `SLACK_WEBHOOK` with the incoming webhook to the current slack channel to recive the notifications

### CODE REVIEW

File: .github/workflows/identify-old-issues-and-pr.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Identify old issues and PR
```
Comment (ThunderSon) [inline .github/workflows/identify-old-issues-and-pr.yml]: I don't think that we still need the python script. I found a stale github actions, but I'd prefer not implementing it yet. There are issues I don't want closed. What would you recommend?

**Comment by ThunderSon** [comment]
Webhook added. To anyone interested in how the hook link was created, you can check the following [incoming webhooks integration](https://slack.com/apps/A0F7XDUAZ-incoming-webhooks)

File: .github/workflows/identify-old-issues-and-pr.yml (hunk 1 above)
Comment (rejahrehim) [reply .github/workflows/identify-old-issues-and-pr.yml]: Yup, We can migrate the python scripts in the next round. I prefer one at a time.

Comment (ThunderSon) [reply .github/workflows/identify-old-issues-and-pr.yml]: This shall be tackled in a different PR

**Pull request merged**